code/game/machinery/cryo.dm

```diff
@@ -451,6 +451,12 @@
 	add_fingerprint(usr)
 	return
 
+/obj/machinery/bodyscanner/narsie_act()
+	go_out()
+	new /obj/effect/gibspawner/generic(get_turf(loc)) //I REPLACE YOUR TECHNOLOGY WITH FLESH!
+	color = "red"//force the icon to red
+	light_color = LIGHT_COLOR_RED

```

Comment:
This is still defined on the path for bodyscanner

Also, is the string "red" valid on the `color` var?